web_src/js/components/RepoActionView.vue

```diff
@@ -425,6 +431,23 @@ const sfc = {
       } else {
         actionBodyEl.append(fullScreenEl);
       }
+    },
+    async hashChangeListener() {
+      const selectedLogStep = window.location.hash;
```

Comment:
What if the hash is empty string

Reply:
Adressed in https://github.com/go-gitea/gitea/pull/25532#discussion_r1243982548.

Reply:
Done in https://github.com/go-gitea/gitea/pull/25532/commits/77c550a97b19b54b3d500c6af1f80faa4227f53b